modules/salt/pages/salt-rate-limiting.adoc

```diff
@@ -71,8 +71,14 @@ Defaults to 1 second.
 == Disabling the Salt Mine
 
 In older versions, {productname} used a tool called Salt mine to check minion availability.
-With the introduction of more efficient tools in {productname}{nbsp}3.2, it is no longer required.
+Salt mine would ping the server every hour, which created significant load.
+With the introduction of more efficient tools in {productname}{nbsp}3.2, Salt mine is no longer required.
+Instead, the {productname} server uses Taskomatic to ping only the minions that appear to have been offline for twelve hours or more, with all minions being pinged only once in every twenty four hour period by default.
```

Comment:
"with all minions being pinged only once in every twenty four hour period by default." -> "with all minions being pinged at least once in every twenty four hour period by default"